File: src/my_cpp_pkg/src/smartphone.cpp
#include "rclcpp/rclcpp.hpp"
#include "example_interfaces/msg/string.hpp"

class SmartPhoneNode : public rclcpp::Node // MODIFY NAME
{
public:
    SmartPhoneNode() : Node("smartphone_cpp") // MODIFY NAME
    {
        subcriber_ = this->create_subscription
                        <example_interfaces::msg::String>("robot_news_cpp", 10, std::bind
                        (&SmartPhoneNode::callbackRobotNews, this, std::placeholders::_1));
        subcriber2_ = this->create_subscription
                        <example_interfaces::msg::String>("robot_sensor_cpp", 10, std::bind
                        (&SmartPhoneNode::callbackRobotNews2, this, std::placeholders::_1));
        RCLCPP_INFO(this->get_logger(),"SmartPhone has been started.");
    }

private:
    void callbackRobotNews(const example_interfaces::msg::String::SharedPtr msg)
    {
        RCLCPP_INFO(this->get_logger(), "%s", msg->data.c_str());
    }
    void callbackRobotNews2(const example_interfaces::msg::String::SharedPtr msg)
    {
        RCLCPP_INFO(this->get_logger(), "%s", msg->data.c_str());
    }

    rclcpp::Subscription<example_interfaces::msg::String>::SharedPtr subcriber_;
    rclcpp::Subscription<example_interfaces::msg::String>::SharedPtr subcriber2_;
};

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto node = std::make_shared<SmartPhoneNode>(); // MODIFY NAME
    rclcpp::spin(node);
    rclcpp::shutdown();
    return 0;
}

File: src/my_py_pkg/my_py_pkg/smartphone.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node

from example_interfaces.msg import String
 
 
class SmartphoneNode(Node): # MODIFY NAME
    def __init__(self):
        super().__init__("smartphone")

        self.subscriber_ = self.create_subscription(String, "robot_news_",self.callback_robot_news,10)
        self.get_logger().info("Smartphone has been started.")

        self.subscriber2_ = self.create_subscription(String, "robot_sensor_",self.callback_robot2_news,10)
        self.get_logger().info("Smartphone has been started.")

    def callback_robot_news(self, msg):
        self.get_logger().info(msg.data)

    def callback_robot2_news(self, msg):
        self.get_logger().info(msg.data)
 
 
def main(args=None):
    rclpy.init(args=args)
    node = SmartphoneNode() # MODIFY NAME
    rclpy.spin(node)
    rclpy.shutdown()
 
 
if __name__ == "__main__":
    main()